platform-coop-toolkit/coop-library
# feat: visual designs

## Description

Implements visual designs from https://files.inclusivedesign.ca/s/mz0ANL9RlXo8ij6.

## Steps to test

1. Visit Home, About, Browse All, Topics, Goals, Co-op Types, Sectors, and Formats.

**Expected behavior:** Implementation should align with designs.

## Additional information

Not applicable.

## Related issues

- Resolves #219.
